command/debug.go

```diff
@@ -55,34 +59,83 @@ General Options:
 
 Debug Options:
 
-  -duration=2m
-   The duration of the log monitor command. Defaults to 2m.
+  -duration=<interval>
```

Comment:
should this be `-duration=<duration>` instead of interval

Reply:
yes! ugh. I'll include that change in the rename pr which I've just rebased on top of this.